testing/bcc/APKBUILD

```diff
@@ -1,22 +1,27 @@
 # Maintainer: Adam Jensen <acjensen@gmail.com>
 pkgname=bcc
 pkgver=0.10.0
-pkgrel=0
+pkgrel=1
 pkgdesc="A toolkit for creating efficient kernel tracing and manipulation programs"
 url="https://github.com/iovisor/bcc/"
 arch="aarch64 x86 x86_64"
 license="Apache-2.0"
 # bcc's test suite requires privileged access to run BPF programs
 options="!check"
 depends="python"
+subpackages="$pkgname-dev:_dev $pkgname-doc:_doc $pkgname-tools:_tools $pkgname-lua:_lua py-$pkgname:_py"
+_libbpfver=0.0.4
```

Comment:
should libbpf be a separate APKBUILD ?

Reply:
I had hoped to simplify the build, but I just noticed that there's been a recent effort to make an RPM package for libbpf. We will probably want one as well.

Force-pushed a change to continue using testing/libbpf-dev.